robyn/openapi.py

```diff
@@ -0,0 +1,271 @@
+import json
+import re
+from dataclasses import dataclass, field, asdict
+from inspect import Signature
+from pathlib import Path
+from typing import List, Dict, TypedDict, Optional
+
+import robyn
+
+
+@dataclass
+class Contact:
+    """
+    The contact information for the exposed API.
+    (https://swagger.io/specification/#contact-object)
+
+    @param name The identifying name of the contact person/organization.
+    @param url The URL pointing to the contact information. This MUST be in the form of a URL.
+    @param email The email address of the contact person/organization. This MUST be in the form of an email address.
```

Comment:
@VishnuSanal , the docstring should also have a type associated to it.
```suggestion
    @param name: Optional[str] The identifying name of the contact person/organization.
    @param url: Optional[str] The URL pointing to the contact information. This MUST be in the form of a URL.
    @param email: Optional[str] The email address of the contact person/organization. This MUST be in the form of an email address.
```